packages/widgetbook_cli/lib/src/commands/build_push.dart

```diff
@@ -154,23 +142,75 @@ class BuildPushCommand extends CliCommand<BuildPushArgs> {
       return 22;
     }
 
-    final zipFile = await zipEncoder.zip(buildDir, '$buildId.zip');
-    if (zipFile == null) {
-      archiveProgress.fail('Failed to create build archive');
-      return 23;
-    }
+    final files = buildDir //
```

Comment:
todo: remove this empty comment

Reply:
It's a trick to have every call on a new line. Otherwise it look as follows, which I find less readable.

```dart
final files = buildDir.listSync(recursive: true).whereType<File>();
```